### PR TITLE
[teswiz-upgrade] Hybrid App Compatibility of Context Switching utility for Appium 2.0 with documentation

### PR DESCRIPTION
**Pull request by ngourkar**
This PR is intended to implement the test scenario for utilities of context switching in "Getting started with teswiz"
-- Scenario and its implementation
--Updated Read me file
--Updated Gradle, config and capability file

-The scenario covers the accessibility of web element embedded into the android app by switching the context to web and then again accessing the native app elements by re-switching the context to native app

### CODE REVIEW

File: caps/vodqa_local_capabilities.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "android": {
+    "adbExecTimeout": 45000,
+    "app": "https://github.com/anandbagmar/sampleAppsForNativeMobileAutomation/raw/main/TheApp-release.apk",
```
Comment (anandbagmar) [inline caps/vodqa_local_capabilities.json:4]: point to vodqa.apk

File: documentation/README-inProgress.md [hunk 2]
```diff
@@ -23,6 +23,14 @@ There are some method name changes as listed below:
 | Purpose                                          | ![#f03c15](https://placehold.co/15x15/f03c15/f03c15.png) Old ![#f03c15](https://placehold.co/15x15/f03c15/f03c15.png) | ![#c5f015](https://placehold.co/15x15/c5f015/c5f015.png) New ![#c5f015](https://placehold.co/15x15/c5f015/c5f015.png) |
 |:-------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------|
 
+## Minor Enhancement
+
+1. setWebViewContext() - Have updated use of AppiumDriver to SupportsContextSwitching for switching to Web View context.
+   Added logger for enhanced logging of context handles.
```
Comment (anandbagmar) [inline documentation/README-inProgress.md:29]: No need to mention logger

File: build.gradle [hunk 3]
```diff
@@ -3,7 +3,8 @@ import java.text.SimpleDateFormat
 buildscript {
     ext {
         gradleVersion = '7.3.3'
-        teswizVersion = '56413e91b0'
+        teswizVersion = 'd97e6840f8'
+        atdVersion = '54ff762128'
```
[the remark this answers is not in the document]
Comment (ngourkar) [reply build.gradle]: updated

File: build.gradle [hunk 4]
```diff
@@ -21,6 +22,7 @@ version '0.0.1'
 repositories {
     mavenLocal()
     maven { url 'https://jitpack.io' }
+    mavenCentral()
```
[the remark this answers is not in the document]
Comment (ngourkar) [reply build.gradle]: updated

File: build.gradle [hunk 5]
```diff
@@ -35,6 +37,7 @@ dependencies {
     implementation("com.github.znsio:teswiz:${project.teswizVersion}") {
         transitive = false
     }
+    implementation "com.github.AppiumTestDistribution:AppiumTestDistribution:$project.atdVersion"
```
[the remark this answers is not in the document]
Comment (ngourkar) [reply build.gradle]: updatd

File: caps/vodqa_local_capabilities.json (hunk 1 above)
Comment (ngourkar) [reply caps/vodqa_local_capabilities.json]: updated

File: documentation/README-inProgress.md (hunk 2 above)
Comment (ngourkar) [reply documentation/README-inProgress.md]: updated

**Pull request merged**